tsengcoin-core/src/v1/request.rs

```diff
@@ -217,6 +218,7 @@ pub fn download_latest_blocks(state_mut: &Mutex<State>) -> Result<(), Box<dyn Er
             },
             _ => {
                 // TODO: Remove node and try again
+                state.network.remove(best_node.to_owned());
```

Comment:
I get an error about borrowing `state.network` as mutable here because it was borrowed as immutable in line 151 - also we should try the request again if one of the nodes returns nonsense

Reply:
Ah I see, just removed this line, lmk if that's okay.